tests/testthat/test-as_reporting_triangle.R

```diff
@@ -362,3 +362,20 @@ test_that("assert on reporting triangle works as expected", {
   rep_tri5$structure <- rep(6, ncol(rep_tri$reporting_triangle_matrix))
   expect_error(assert_reporting_triangle(rep_tri5))
 })
+
+test_that("`as_reporting_triangle()` appropriately warns if there is nothing to be nowcasted (no unobserved cases in reporting triangle)", { # nolint
```

Comment:
its info now?

Reply:
Yes its `cli_alert_info` and so I changed this to say `appropriately messages`